rules/windows/defense_evasion_scheduledjobs_at_protocol_enabled.toml

```diff
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/11/23"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempts to enable the Windows Scheduled Jobs AT protocol. Attackers may use this method to move laterally or
+persist locally via the AT Jobs Protocol, which is bound to deprecation starting with Windows 8 and Windows Server 2012.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Scheduled Jobs AT Protocol Enabled"
+references = [
+    "https://docs.microsoft.com/en-us/windows/win32/cimwin32prov/win32-scheduledjob?redirectedfrom=MSDN",
+]
+risk_score = 47
+rule_id = "9aa0e1f6-52ce-42e1-abb3-09657cee2698"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+type = "eql"
```

Comment:
Might be better to use KQL for simpler queries, unless there is a specific reason for EQL

Reply:
any technical reasons ?

Reply:
@brokensound77 is there a performance or other blocker to using EQL vs KQL?